src/pages/projectSettings/tabs/NotificationsTab/transformers.ts

```diff
@@ -149,15 +153,18 @@ export const gqlToForm: GqlToFormFunction<Tab> = (data) => {
 };
 
 export const formToGql: FormToGqlFunction<Tab> = (
-  { buildBreakSettings }: FormState,
+  { buildBreakSettings, subscriptions }: FormState,
   id
 ) => {
   const projectRef: ProjectInput = {
     id,
     notifyOnBuildFailure: buildBreakSettings.notifyOnBuildFailure,
   };
-  // TODO in EVG-16971
+  const transformedSubscriptions: SubscriptionInput[] = subscriptions.map(
+    (subscription) => getGqlPayload(subscription)
+  );
```

Comment:
[nit] fancy syntax 😛 
```suggestion
  const transformedSubscriptions: SubscriptionInput[] = subscriptions.map(getGqlPayload);
```